### ISSUE TITLE
FYI: Official GitHub feature now being rolled out

### ISSUE THREAD

**Issue by hftf** [marpo60/github-subscribed#7]
See https://github.com/isaacs/github/issues/283 https://help.github.com/en/articles/listing-the-issues-and-pull-requests-youre-subscribed-to

**Comment by aspiers** [comment]
Yeah - I think it probably makes sense to retire this extension now.
